docs/userGuide/siteJsonFile.md

```diff
@@ -253,3 +256,16 @@ The date format is thus - <br>
   <include src="pages/locales.md" />
 </panel>
 <br>
+
+#### **`intrasiteLinkValidation`**
+
+**Toggle whether to validate intrasite links.** By default, Markbind will validate all intrasite links and alert you of any potentially invalid ones. 
+You can choose to turn this option off to not receive warnings during the `build` process. 
```

Comment:
```suggestion
To disable this validation **entirely**, you may add the following to `site.json`:
```

`build` sounds a little technical 🤔

Reply:
Plus, it shouldn't be in `code` font?
intrasite or intra-site?
Markbind -> MarkBind

Reply:
> build sounds a little technical 🤔
> Plus, it shouldn't be in code font?
Hmm I used `build` it was originally used for `disableHtmlBeautify`. But I have removed it from both sides in my commit. 

> intrasite or intra-site?
For intrasite vs intra-site, I think I will stick to intra-site in the documentation (since it seems to be the consistent throughout). 

> Markbind -> MarkBind
Ah, right. Thanks for pointing this out!